Kernel/Arch/aarch64/vector_table.S

```diff
@@ -6,7 +6,9 @@
 
 .section .text.vector_table
 
-#define REGISTER_STATE_SIZE 264
+// NOTE: This size must be a multiple of 16 bytes, to ensure that the stack pointer
+//       stays 16 byte aligned.
+#define REGISTER_STATE_SIZE 272
```

Comment:
Can we compile time check this with

```
#if REGISTER_STATE_SIZE % 16 != 0
#error noooo
#endif
```

?

Reply:
We can do that, and in fact I just pushed a commit that actually does this. Thanks for the recommendation!